staging/src/k8s.io/apimachinery/pkg/test/BUILD

```diff
@@ -0,0 +1,52 @@
+package(default_visibility = ["//visibility:public"])
+
+licenses(["notice"])
+
+load(
+    "@io_bazel_rules_go//go:def.bzl",
+    "go_library",
+    "go_test",
+)
+
+go_test(
+    name = "go_default_test",
+    srcs = [
+        "api_meta_help_test.go",
+        "api_meta_meta_test.go",
+        "apis_meta_v1_unstructed_unstructure_test.go",
+        "runtime_helper_test.go",
+        "runtime_serializer_protobuf_protobuf_test.go",
+        "runtime_unversioned_test.go",
+    ],
+    library = ":go_default_library",
+    tags = ["automanaged"],
+    deps = [
+        "//vendor/github.com/google/gofuzz:go_default_library",
+        "//vendor/k8s.io/api/core/v1:go_default_library",
```

Comment:
This is causing import cycle. I don't know why our [script](https://github.com/kubernetes/kubernetes/blob/master/hack/verify-staging-imports.sh#L68) didn't catch this.